onnx/backend/test/case/node/thresholdedrelu.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class ThresholdedRelu(Base):
+
+    @staticmethod
+    def export():
```

Comment:
Since this code snippet also serves as docs, could you also add some case which contains manually created input and output. This is very helpful for users to understand the op spec.

Reply:
https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/ceil.py#L23

Just like this

Reply:
Done. Please review.